jenkins/promotion/release-promotion.jenkinsfile

```diff
@@ -49,378 +49,388 @@ pipeline {
+                                                                                            string(name: 'DISTRIBUTION_PLATFORM', value: 'linux'),
+                                                                                            string(name: 'DISTRIBUTION_NAME', value: 'tar'),
+                                                                                            string(name: 'DISTRIBUTION_ARCHITECTURE', value: 'arm64'),
+                                                                                            string(name: 'DISTRIBUTION_BUILD_NUMBER', value: params.OPENSEARCH_DASHBOARDS_RC_BUILD_NUMBER),
+                                                                                            string(name: 'INPUT_MANIFEST', value: "${params.RELEASE_VERSION}/opensearch-dashboards-${params.RELEASE_VERSION}.yml"),
+                                                                                        ]
+                        echo 'Promotion successful for OpenSearch Dashboards Linux tar arm64!'
+                    }
                 }
             }
-            steps {
-                echo 'Triggering distribution-promote-artifacts for OpenSearch Dashboards Linux tar arm64'
-                build job: 'distribution-promote-artifacts', wait: true, parameters: [string(name: 'DISTRIBUTION_JOB_NAME', value: 'distribution-build-opensearch-dashboards'), 
-                                                                                    string(name: 'DISTRIBUTION_PLATFORM', value: 'linux'),
-                                                                                    string(name: 'DISTRIBUTION_NAME', value: 'tar'),
-                                                                                    string(name: 'DISTRIBUTION_ARCHITECTURE', value: 'arm64'),
-                                                                                    string(name: 'DISTRIBUTION_BUILD_NUMBER', value: params.OPENSEARCH_DASHBOARDS_RC_BUILD_NUMBER),
-                                                                                    string(name: 'INPUT_MANIFEST', value: "${params.RELEASE_VERSION}/opensearch-dashboards-${params.RELEASE_VERSION}.yml"),
-                                                                                ]
-                echo 'Promotion successful for OpenSearch Dashboards Linux tar arm64!'
-            }
         }
-        stage('OpenSearch Dashboards Linux tar x64') {
-            agent {
-                docker {
-                    label AGENT_LINUX_X64
-                    image 'docker/library/alpine:3'
-                    registryUrl 'https://public.ecr.aws/'
-                    alwaysPull true
+    }
+    post {
+        success {
+            stage('Triggering X64 Promotion Jobs') { 
+                parallel {
+                    stage('OpenSearch Dashboards Linux tar x64') {
```

Comment:
I believe we still want this to be promoted within workflow just not in parallel. I think we can add this after parallel{} block right?